### DISCUSSION TITLE
写笔记时直接链接？

### DISCUSSION THREAD

**Discussion by XeonDoer** [windingwind/zotero-better-notes#904] [Ideas]
不知道作者能否增加这样的功能？
1. 一方面，我想在编辑的时候能通过如 @ 这样的字符实现快速的链接到已有的文献、笔记甚至注释上，而不是现在需要到想链接的笔记上点betternote的符号才行。
2. 另一方面，当前只能一次选一个主笔记，我想在任何一个笔记上都能实现这样的功能，避免切换的麻烦。
如果能实现的话，就可以在zotero上做到大纲上的写作，甚至是论文的一个草稿版，再导出时就方便后续的写作。不知道现在有没有这样的插件？如果链接到 obsidian 或 notion 等外面的APP 还要单独导来导去的感觉很麻烦

**Comment by LeoMarsX** [comment]
同意，期待实现在zotero上可以完成论文初稿。
